localization/languages/de.json

```diff
@@ -135,7 +135,10 @@
             "unsafeHTML": null //missing translation
         },
         "settingsAppearanceHeading": "Aussehen",
-        "settingsDarkModeToggle": "Dark-Mode einschalten",
+        "settingsEnableDarkMode": "Dunkelmodus einschalten:", 
+        "settingsDarkModeNever": "Nein", 
+        "settingsDarkModeNight": "In der Nacht", 
+        "settingsDarkModeAlways": null, //"Always" missing translation
```

Comment:
Would it be possible to translate this as well?

Reply:
Oh, yes. Didn't notice